src/mem/cache/tags/partitioning_policies/partitionFieldsExtension.hh

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
Could you create a .cc and move the implementation there?

Also, I think that we typically add a space after {, and before } in single-line function declarations.